.travis.yml

```diff
@@ -16,10 +16,10 @@ script: "./.travis.sh"
 jobs:
   include:
     - stage: GitHub Release
-      if: tag IS present
-      d: ldc
+      #if: tag IS present
```

Comment:
Apparently building the binaries on Travis is vastly more complicated than thought, so let's better constantly build them to avoid bad surprises on releases. Imho the ~1.5min are worth the mind peace.